internal/pkg/manifest/workload_test.go

```diff
@@ -19,14 +18,43 @@ func TestImage_UnmarshalYAML(t *testing.T) {
 		inContent []byte
 
 		wantedError error
+		wantImage   Image
 	}{
-		"error if both build and location are set": {
```

Comment:
i might be missing something! Do we not need this test anymre?

Reply:
It's not necessary to check this when unmarshaling the YAML because we check it shortly afterwards in `Image.validate`! This test doesn't take that into account.